cmd/flags.go

```diff
@@ -155,12 +161,37 @@ func fileFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 }
 
 func pathFilterFlags(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
-	cmd.Flags().String(flagFilterRule, "", `filter rule ("allowlist", "denylist", or "" for no filtering)`)
-	if err := v.BindPFlag(flagFilterRule, cmd.Flags().Lookup(flagFilterRule)); err != nil {
+	flags := cmd.Flags()
+	flags.String(flagFilterRule, "blank", `filter rule ("allowlist", "denylist", or "" for no filtering)`)
```

Comment:
Just a suggestion but `blank` could be a defined constant since we reference it a few times in `paths.go` as well. This would eliminate the need to hunt down other references in the off chance that this value changes in the future.

Reply:
good call, updated